### PR TITLE
Keep this repository in a buildable condition

### PR DESCRIPTION
**Pull request by ctrlcctrlv**
There's an issue from last year, #204, which @probonopd and @skef primarily contributed to.

This continues on that…it switches the build from gitbook to honkit (fork of gitbook because gitbook is deprecated and no longer supported by nodeJS, or at least, I can't get it to install anymore with `npm`).

It also makes minor changes to CSS so that #211 looks better.

There were minor reasons why #204 wasn't closed w/a CI workflow, basically, the generated books don't exactly match old books. I copied what I could from @probonopd's branch—the other bugs honestly should just be left as is, right now, having such an old version of the book is way worse than a version with a few gray boxes on some pages.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,5 @@ _site/
 .DS_Store
 node_modules
 ebook/temp/**
+.jekyll-cache/
+package-lock.json
```
Comment (jtanx) [inline .gitignore]: I thought you were meant to commit the package lock

Comment (ctrlcctrlv) [reply .gitignore]: It's not in the repository at present, so, no? Committing the package lock would only guarantee lower versions of dependencies be used, which may be insecure (but perhaps less prone to regressions)

Comment (jtanx) [reply .gitignore]: I would not be surprised if this repo predated when package-lock.json was introduced to npm and hence why it's not committed.

Everything I can find online suggests you should commit it, I think the lockfile isn't used unless you use a specific command (`npm ci`?) although not sure what GH does.

File: ebook/create_books.sh [hunk 2]
```diff
@@ -1,19 +1,29 @@
 #!/bin/bash
 #ebooks creation script v2.0
 
+HONKIT="$PWD/node_modules/honkit/bin/honkit.js"
```
Comment (jtanx) [inline ebook/create_books.sh]: You might be able to do `npx honkit` instead

**Review by jtanx — APPROVED** [review]

**Pull request merged**